package.json

```diff
@@ -67,7 +68,7 @@
     "fmt:gql": "prettier --write \"**/*.graphql\"",
     "git:info": "atomist git",
```

Comment:
This is `atm-git-info` or something like that now.

Reply:
Replaced.